### PR TITLE
docs: add warning about packageManager config

### PR DESCRIPTION
**Pull request by capaj**
just something I bumped into today

image

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
I suspect we should be able to infer the package manager in use??

**Comment by Cherry** [comment]
@petebacondarwin This is inferred by https://github.com/cloudflare/wrangler-action/pull/193, and I'm not really sure the "default" copy/pasted config should use `pnpm` - that's definitely not the standard.

> "The package manager you'd like to use to install and run wrangler. If not specified, the preferred package manager will be inferred based on the presence of a lockfile or fallback to using npm if no lockfile is found. Valid values are `npm` | `pnpm` | `yarn` | `bun`."

You do not need to specify `packageManager` as the docs now suggests, as long as you're running an up to date version.

**Comment by Cherry** [comment]
The detection of `pnpm` looks for a `pnpm-lock.yaml` file in the working directory, but only after looking for a `package-lock.json`, and then a `yarn.lock`.

If you have a `pnpm-lock.yaml` committed, and none of these other files, it should work as expected.

File: README.md [hunk 1]
```diff
@@ -52,6 +53,10 @@ jobs:
 
 ## Configuration
 
+### packageManager
```
Comment (IgorMinar) [inline README.md:56]: Unfortunately this header makes all the following paragraphs appear as if they were part of the packageManager docs.

**Pull request merged**